src/main/java/moneytransferapp/MoneyTransferWorkflowImpl.java

```diff
@@ -0,0 +1,34 @@
+package moneytransferapp;
+
+import io.temporal.activity.ActivityOptions;
+import io.temporal.workflow.Workflow;
+ import io.temporal.common.RetryOptions;
+import java.time.Duration;
+
+// @@@SNIPSTART money-transfer-project-template-java-workflow-implementation
+public class MoneyTransferWorkflowImpl implements MoneyTransferWorkflow {
+
+    RetryOptions retryoptions =  RetryOptions.newBuilder()
+            .setInitialInterval(Duration.ofSeconds(1))
+            .setMaximumInterval(Duration.ofSeconds(100))
+            .setBackoffCoefficient(2)
+            .setMaximumAttempts(500)
+            .build();
+
+    private final ActivityOptions options = ActivityOptions.newBuilder()
+            .setStartToCloseTimeout(Duration.ofSeconds(5))
+            .setRetryOptions(retryoptions)
+            .build();
+
+    // An ActivityStub converts the parameters that are passed to it so that the params can be passed to the server.
+    private final AccountActivity account = Workflow.newActivityStub(AccountActivity.class, options);
```

Comment:
I would dumb the comment down and be more literal. obviously not exactly this but,

"This lets you call activites like local java methods and actually does RPC.."

Reply:
Cool - rewrote.